docs/asset/cfd_trading.puml

```diff
@@ -0,0 +1,50 @@
+@startuml
```

Comment:
Can we move this file into a different repository? It includes information out of scope of this project e.g. `maker_automation`

Reply:
fine with me.. I will move the pull request then.